wit/ephemeral/outbound-redis.wit

```diff
@@ -0,0 +1,13 @@
+use * from redis-types
+
+// Publish a Redis message to the specificed channel and return an error, if any.
+publish: function(address: string, channel: string, payload: payload) -> expected<_, error>
+
+// Get the value of a key. An error is returned if the value stored at key is not
+// a string.
+get: function(address: string, key: string) -> expected<option<string>, error>
```

Comment:
I just realized that we should probably get and set byte arrays as opposed to strings.

Reply:
return as well?

Reply:
Yeah, that's what I was thinking.